LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 NOCWorx L.L.C.
```

Comment:
is spelled "NocWorx".  Need a codesniff for this.

Reply:
Copied that straight out of the docblock at the head of every file.

Reply:
The comment block that's in every PHP file spells it NOCWorx L.L.C.

Reply:
i know (not blaming you). that's old, new stuff should be `NocWorx`.

and the "codesniff" part was mainly a joke  :)

Reply:
done